README.md

```diff
@@ -20,9 +13,23 @@ npm install pcln-design-system
 ## Contributing
 
 If you'd like to contribute to the design system, we'd love to have your help.
-If you work at Priceline.com, please join the #design-system Slack channel and let us know what you'd like to work on.
-
-For all contributors, please be sure to read the [Contributing](CONTRIBUTING.md) doc.
+If you work at Priceline.com, please join the #design-system Slack channel and
+let us know what you'd like to work on.
+
+For all contributors, please be sure to read the
+[Contributing](CONTRIBUTING.md) doc.
+
+Highlight topics:
```

Comment:
With the TOC in the contributing doc, do we need this section? Seems like it might go out of sync over time